tests/handlers/test_errata.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+import json
+from os import path
+
+from estuary.models.errata import Advisory
+
+import requests
+import mock
+from tests import message_dir
+from estuary_updater.handlers.errata import ErrataHandler
+from estuary_updater import config
+import iso8601
+
+
+def test_errata():
+    """Test the errata handler when it recieves a new message."""
+    with open(path.join(message_dir, 'errata', 'errata_api.json'), 'r') as f:
+        errata_api_msg = json.load(f)
+    url = 'https://errata.devel.redhat.com/api/v1/erratum/34661'
+    with mock.patch('requests.get') as mock_get:
+        mock_response = mock.Mock()
+        mock_response.json.return_value = errata_api_msg
+        mock_get.return_value = mock_response
+        requests.get(url).json()
+        with open(path.join(message_dir, 'errata', 'activity_status.json'), 'r') as f:
+            msg = json.load(f)
+        assert ErrataHandler.can_handle(msg) is True
+        handler = ErrataHandler(config)
+        handler.handle(msg)
+        advisory = Advisory.nodes.get_or_none(id_='34661')
+        assert advisory is not None
+
+        assert advisory.actual_ship_date is None
+        assert advisory.advisory_name == 'RHEA-2018:34661-01'
+        assert advisory.content_types == ['rpm']
+        assert advisory.created_at == iso8601.parse_date('2018-06-15T15:26:38Z')
```

Comment:
Optional: You should create an actual datetime object manually to test against. Then it'd allow you to catch bugs in the library or how you're using the library.